lib/client.js

```diff
@@ -357,22 +391,42 @@ extend(Raven.prototype, {
       res.sentry = eventId;
       return next(err);
     };
+  },
+
+  captureBreadcrumb: function (breadcrumb) {
+    if (!this.installed) return;
```

Comment:
This was a spot where I thought about leaving a comment but didn't; I'll add one. The purpose here is to avoid capturing breadcrumbs halfway through instrumentation when we're not in a context yet. We do a consoleAlert for each instrumentation, so consider if console instrumentation happens first and then http instrumentation. Without this early exit condition we would try to capture a breadcrumb for the http instrumentation's consoleAlert.